eduiddashboard/views/nins.py

```diff
@@ -195,7 +195,7 @@ def send_letter(request, user, nin):
                 'Please try again later.')
     if response.status_code == 200:
         expires = response.json()['letter_expires']
-        expires = datetime.utcfromtimestamp(expires)
+        expires = datetime.utcfromtimestamp(float(expires))
```

Comment:
What am I missing? Floats are usually best avoided if you ask me ;)

>>> datetime.datetime.utcfromtimestamp(int(time.time()))
datetime.datetime(2015, 12, 18, 9, 50, 54)
>>>